content/en/reference/general/overview/index.md

```diff
@@ -185,7 +185,7 @@ For running a node cluster of load balanced Lotus JSON RPC API endpoints.
 - [Codefi Storage](https://storage.codefi.network/) - view the Filecoin Storage Market, storage provider info, asking price, completed deals.
 - [FIL Swan](https://www.filswan.com/) - for offline deals, storage provider info, prices, offline deal acceptance.
 - [Filrep.io](https://filrep.io/) - Index of online storage providers and their pricing, ranked by power and reputation score.
-- [Textile Storage Provider Index](https://blog.textile.io/introducing-the-miner-index/) - API and CLI to find storage providers by price, observed deals, speed from North American nodes.
+- [Textile storage provider Index](https://blog.textile.io/introducing-the-miner-index/) - API and CLI to find storage providers by price, observed deals, speed from North American nodes.
```

Comment:
"Textile storage provider Index" should either be "Textile Storage Provider Index" or "Textile storage provider index" but since the actual site has long since bitrotted- https://minerindex.hub.textile.io/docs/ - I suggest deleting it altogether.

Reply:
Yeah let's take this link out. It's not loading for me.